src/node_file.cc

```diff
@@ -3047,11 +3047,11 @@ void BindingData::LegacyMainResolve(const FunctionCallbackInfo<Value>& args) {
     return;
   }
 
+  std::string error_invalid_module_path =
+      "Cannot find entry file for module " + module_path + " in " + module_base;
```

Comment:
Perhaps something like:

```suggestion
      "No package entry point defined for package " + module_path + " imported from " + module_base;
```